Microsoft.Azure.Functions.PowerShellWorker.Common.props

```diff
@@ -0,0 +1,11 @@
+<Project>
```

Comment:
Non-blocking. Maybe the file name can be simplified a bit? like `PowerShellWorker.Common.props`.

Reply:
Fixed